src/data_provider/src/packages/packageLinuxDataRetriever.h

```diff
@@ -88,10 +87,11 @@ class FactoryPackagesCreator<LinuxType::STANDARD> final
                 getRpmInfo(callback);
             }
 
-            if (Utils::existsDir(APK_PATH))
+            if (Utils::existsDir(SNAP_PATH))
             {
-                getApkInfo(APK_DB_PATH, callback);
```

Comment:
Careful during rebases, here we are modifying other developments.

Reply:
rebases with inactive projects for so long are always tricky, even using `-X theirs` switches, I'll check this out

Reply:
I'd avoid the use of `-X theirs` and resolve manually every conflict to inspect the code and avoid any silent code add/removal

Reply:
I didn't know that option exist, but that is exactly the issue. When doing a rebase "theirs" is your implementation because you are in the branch to rebase against (master in this case). So it will always ignore what is master (in this case) branch and will replace its content with the new code you are adding.
I do what @pereyra-m does.  It gives you more control and I think is the way to go, but I can't agree more about rebasing old branches, it is always tedious.

Reply:
anyway, whether with `-X theirs` or not, checking with the base branch is the key